assets/src/edit-story/components/metaBoxes/index.js

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
I wonder if it would be good to create an integrations folder to separate the WP-specific logic. WDYT?